backend/app/core/security.py

```diff
@@ -32,3 +32,7 @@ def create_access_token(*, data: dict, expires_delta: Optional[timedelta] = None
     to_encode.update({"exp": expire})
     encoded_jwt = jwt.encode(to_encode, SECRET_KEY, algorithm=ALGORITHM)
     return encoded_jwt
+
+
+def get_password_reset_token_expiry_ts() -> datetime:
+    return datetime.utcnow() + timedelta(weeks=1)
```

Comment:
Given that we're sending this by email, which isn't exactly known for security, a reset token valid for a week seems like a very long time.

Reply:
Oops, it's missing the "make configurable" TODO. I might as well just add it in now.